### ISSUE TITLE
About training log and the final loss

### ISSUE THREAD

**Issue by Wp-Zhang** [codeslake/Color_Transfer_Histogram_Analogy#16]
Hi! Your work is impressive and really interesting.

I'm trying to write the training code myself as the training code is not released, but I don't really know if I'm doing it right. The training takes time and I can't wait till the end of training to check whether my code is correct. 

Do you still have the training log, or do you remember the training loss after the 100 epochs of training so that I can use them as a reference? It would be really helpful if you could share the code for training. Thanks!

**Comment by codeslake** [comment]
Hi, Zhang.

I don't have access to the training code at the moment as our lab is doing a major NAS upgrade.
I don't remember the exact epoch #, but I do remember it took quite a time to converge (I think it was over a week or so).

**Comment by Wp-Zhang** [comment]
Thanks for your reply! I would appreciate it if you could share the code after the upgrade 😃

**Comment by codeslake** [comment]
I am afraid that I can share the code fast enough (I am preparing for my doctoral defense and other things for my next career).
I will have to organize all the works that I have done during my Ph.D. before graduation. That will be the moment the training code can be shared in this repo. I am expecting it to be on Feb 2023.  
Anyway, thanks for bringing this to my attention! 
Best,
